### DISCUSSION TITLE
Facing the problem to get IP and country details of yourls generated shorten link

### DISCUSSION THREAD

**Discussion by tbs-chirag** [YOURLS/YOURLS#3700] [Q&A]
Hi,

I use yourls shorten last couple of time and that is awesome. I have one doubt for stat functionality.

Is there any possible way where I can get country and IP address list for particular clicked shorten URL in yourls.
I can get it when I do login and click stats icon from yourls admin panel. 

but I want the same details via API. please help me about how can I get it from there.

Thank You!

[image_2024_01_03T07_47_11_303Z](https://github.com/YOURLS/YOURLS/assets/131744449/f4bec686-bf3f-49d7-b20e-31f00d4265ea)

**Comment by ozh** [comment]
See https://yourls.org/readme.html#API

**Reply by tbs-chirag** [reply]
Hi @ozh , is it possible to get multiple list of countries and IP addresses of clicked link? I read the provided documentation but not able to find the way....

please help me !!

**Reply by tbs-chirag** [reply]
Hi @ozh , any solution or suggestion you recommend?

**Reply by ozh** [reply]
Probably coding a custom API action (there are example plugins for this) and polling the log table for data
